packages/jellyfish-api-core/__tests__/category/governance/createCfp.test.ts

```diff
@@ -0,0 +1,138 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import BigNumber from 'bignumber.js'
+import { UTXO, ProposalStatus, ProposalType } from '../../../src/category/governance'
+import { RpcApiError } from '@defichain/jellyfish-api-core'
+
+describe('Governance withthout WalletCoinbaseMaturity', () => {
+  const container = new MasterNodeRegTestContainer(undefined, 'defi/defichain:HEAD-a61f615')
+  const client = new ContainerAdapterClient(container)
+  let payoutAddress: string
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+
+    payoutAddress = await container.call('getnewaddress')
```

Comment:
don't declare global/shared variable when you don't have to, it makes test code harder to read as you need to carry that `let` in your mind creating greater cognitive load when others read your code.